api/app/Console/Commands/CreateUserCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Database\Doctrine\EntityManager;
+use App\Entities\User;
+use App\Enum\Role;
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+
+class CreateUserCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'user:create {--name=} {--email=} {--moderator} {--contributor}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a user depending on the role';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle(EntityManager $em): bool
+    {
+        // Get variables from input scrypt
+        $name = $this->option('name');
+        $email = $this->option('email');
+        $moderator = $this->option('moderator');
+        $contributor = $this->option('contributor');
+        $role = null;
+
+        // Check to see if a role has been passed
+        if (!$moderator and !$contributor) {
+            $this->info('Moderator or Contributor was not passed');
+            return false;
+        }
+        if ($moderator and $contributor) {
+            $this->info('Moderator and Contributor were both passed. Please pass only one');
+            return false;
+        }
+
+        // Assign $role the correct information
+        if ($moderator) {
+            $role = Role::MODERATOR();
+        } else if($contributor) {
+            $role = Role::CONTRIBUTOR();
+        }
```

Comment:
This whole section can be simplified if we just default to `contributor` role unless the `--moderator` flag is passed.

Just a suggestion.

```suggestion
        $role = $this->option('moderator') ? Role::MODERATOR() : Role::CONTRIBUTOR();
```